package/gluon-web-admin/luasrc/lib/gluon/web/model/admin/remote.lua

```diff
@@ -43,12 +51,20 @@ local s = f_password:section(Section)
 
 local pw1 = s:option(Value, "pw1", translate("Password"))
 pw1.password = true
+pw1.optional = true
+pw1.datatype = mintype
 function pw1.cfgvalue()
 	return ''
 end
 
-local pw2 = s:option(Value, "pw2", translate("Confirmation"))
+local hint = nil;
```

Comment:
Unneeded semicolon and initialization (a variable's value will default to nil when not specified)